.ci/tweet-releasenotes.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+set -e
+
+source ./.ci/util.sh
+
+checkForVariable() {
+  if [[ "$(declare -p "$1")" =~ ^declare\ -x ]]; then
```

Comment:
can we use `-v` ?

see setting.xml file for reference ?

Reply:
Changes Done